realm/src/main/java/io/realm/internal/Mixed.java

```diff
@@ -68,51 +69,31 @@ public Mixed(byte[] value) {
         this.value = value;
```

Comment:
The definition of value is placed in the middle of the file. Would be nice if it was moved up where it belongs.